syntax = "proto3";

// import "google/protobuf/timestamp.proto";
option go_package = "./;pb";


service SpeechService {
    rpc SpeechToText(SpeechToTextRequest) returns (SpeechToTextResponse);
}

message SpeechToTextRequest {
    bytes audio = 1;
}

message SpeechToTextResponse {
    string text = 1;
}